package.json

```diff
@@ -72,7 +72,9 @@
     "fast-csv": "^3.4.0",
     "fast-xml-parser": "^3.12.20",
     "file-saver": "^2.0.2",
+    "generate-password": "^1.4.2",
     "i18next": "^17.0.11",
+    "i18next-node-fs-backend": "^2.1.3",
```

Comment:
do we still need this?

Reply:
See answer below, needed if we load the language at server startup

Reply:
Switched so i18nFactory so not needed